main.go

```diff
@@ -150,12 +155,9 @@ func parseSuperKeyDestroyRequest(value []byte) (*superkey.DestroyRequest, error)
 	return &request, nil
 }
 
-// getEventType - iterates through headers to find the `event_type` header
-// from miq-messaging.
-// returns: event_type value
-func getEventType(headers []kafka.Header) string {
+func getHeader(name string, headers []kafka.Header) string {
 	for _, header := range headers {
-		if header.Key == "event_type" {
+		if header.Key == name {
```

Comment:
For my knowledge - you're just keying off the fact that if `header.Key == name` - Why is that significant?

Reply:
so the `kafka.Header` struct has 2 fields, [Key/Value](https://github.com/segmentio/kafka-go/blob/6345479c17e8efaca217ccf53ea7b5ea38410b25/protocol/record.go#L49-L52). 

The name is just the value of the Key field in the header, which gives us a value. It's kind of like using the Key as a hash key and the `[]Header` is the hash.

Reply:
To add some history, I had a helper method that just got the value of the `"event_type"` header before, but since I have multiple places I needed to get headers now I just made it a method that fetches a header value based on a name.